File: Java/프로그래머스/0/120891. 369게임/369게임.java
class Solution {
    public int solution(int order) {
        int answer = 0;
        int count=0;
        String str = Integer.toString(order);
        char ch[] = str.toCharArray();
        for(int i=0; i<ch.length;i++){
            if(ch[i]=='3' ||ch[i]=='6'||ch[i]=='9') count++;
        }
        answer=count;
        return answer;
    }
}

File: 프로그래머스/0/120891. 369게임/369게임.js
function solution(order) {
    let answer = 0;
    let a=String(order);
    for(let i=0 ; i<a.length ; i++){
        if(a.charAt(i)=='3' ||a.charAt(i)=='6' ||a.charAt(i)=='9') answer++;
    }
    return answer;
}